tests/symbolicator/test_minidump_full.py

```diff
@@ -120,38 +114,21 @@ def test_full_minidump_invalid_extra(self):
 
         with self.feature("organizations:event-attachments"):
             with open(get_fixture_path("windows.dmp"), "rb") as f:
-                resp = self._postMinidumpWithHeader(
-                    f, {"sentry": "{{{{", "foo": "bar"}  # invalid sentry JSON
+                event = self.post_and_retrieve_minidump(
+                    {"upload_file_minidump": f},
+                    {"sentry": "{{{{", "foo": "bar"},  # invalid sentry JSON
                 )
-                assert resp.status_code == 200
-                event_id = resp.content
 
-        event = eventstore.get_event_by_id(self.project.id, event_id)
         assert not event.data.get("logger")
         assert event.data.get("extra") == {"foo": "bar"}
         # Other assertions are performed by `test_full_minidump`
 
-    def test_raw_minidump(self):
-        self.project.update_option("sentry:store_crash_reports", STORE_CRASH_REPORTS_ALL)
-        self.upload_symbols()
-
-        with self.feature("organizations:event-attachments"):
-            with open(get_fixture_path("windows.dmp"), "rb") as f:
-                # Send as raw request body instead of multipart/form-data
-                resp = self._postMinidumpWithHeader(f, raw=True)
-                assert resp.status_code == 200
-                event_id = resp.content
-
-        event = eventstore.get_event_by_id(self.project.id, event_id)
-        insta_snapshot_stacktrace_data(self, event.data)
```

Comment:
Removing this test is OK. This is now tested here:

https://github.com/getsentry/relay/blob/6ffd86d32c030c27bcf5169f836bf03d51e1aca7/tests/integration/test_minidump.py#L247